tests/appsec/waf/test_rules.py

```diff
@@ -309,7 +309,7 @@ class Test_DiscoveryScan:
     """AppSec WAF Tests on Discovery Scan rules"""
 
     def setup_security_scan(self):
-        self.r1 = weblog.get("/etc/")
+        self.r1 = weblog.get("/etc/something")
```

Comment:
Next makes automatic redirect when the url is finished by `/`, and the rule says that path should contain `/etc/` and not `/etc`, so I modified the url to prevent finishing `/` and also contain `/etc/`.

Reply:
Maybe it's possible to toggle that feature off in next settings ? it's a bit weird to change a test just because one node framework is a bit stupid.

Reply:
I can't find the config property for it...

Reply:
Yes, It is configurable... but

> By default Next.js will redirect urls with trailing slashes to their counterpart without a trailing slash. For example /about/ will redirect to /about. You can configure this behavior to act the opposite way, where urls without trailing slashes are redirected to their counterparts with trailing slashes.

It can be configured to redirect from `path/` to `path` or from `path` to `path/`